gbpservice/nfp/configurator/drivers/loadbalancer/v1/haproxy/haproxy_lb_driver.py

```diff
@@ -0,0 +1,791 @@
+# Copyright 2013 New Dream Network, LLC (DreamHost)
```

Comment:
This looks like a different license. Please use the same license that was used in other files.